tests/platform_tests/test_advanced_reboot.py

```diff
@@ -130,3 +131,39 @@ def test_cancelled_warm_reboot(request, add_fail_step_to_reboot, verify_dut_heal
     add_fail_step_to_reboot('warm-reboot')
     advancedReboot = get_advanced_reboot(rebootType='warm-reboot', allow_fail=True)
     advancedReboot.runRebootTestcase()
+
+
+def test_service_warm_restart(request, duthosts, rand_one_dut_hostname, verify_dut_health, get_advanced_reboot,
+    advanceboot_loganalyzer, capture_interface_counters):
+    duthost = duthosts[rand_one_dut_hostname]
+
+    feature_list = duthost.show_and_parse('show feature status')
+    ignored_services = request.config.getoption("--ignore_service")
+    candidate_service_list = []
+    for feature_data in feature_list:
+        if feature_data['feature'] in ['database', 'syncd']:
+            # Features that do not support warm restart
+            continue
+
+        if feature_data['setowner'] == 'local':
+            # Currently, local feature is not supported for warm restart
+            continue
```

Comment:
setowner == local means it is locally managed docker container (started and managed by systemd), while setowner == kube means it is a container started and managed by kubernetes. I think the intention was to check if it is "kube" then skip it ?

Reply:
will fix